packages/patternfly-4/react-core/src/components/Dropdown/DropdownToggleCheckbox.test.tsx

```diff
@@ -35,13 +40,3 @@ test('passing HTML attribute', () => {
   );
   expect(view).toMatchSnapshot();
 });
-
-test('checkbox passes value and event to onChange handler', () => {
```

Comment:
I had to remove this test because jest is not playing nicely with mount and `currentTarget` https://github.com/airbnb/enzyme/issues/218.

I had to use mount because checkbox is now in `Checkbox` component.

Reply:
I've run into this same issue (I think also with Checkbox). My solution was to change `handleChange` in `Checkbox.tsx` from:
```ts
private handleChange = (event: React.FormEvent<HTMLInputElement>): void => {
  this.props.onChange(event.currentTarget.checked, event);
}
```
to:
```ts
private handleChange = (event: React.FormEvent<HTMLInputElement>): void => {
  this.props.onChange(event.target.checked, event);
}
```

If Checkbox doesn't break, feel free to try that.

Reply:
I think we should open a high-priority issue to add this test back if we aren't willing to address it in the scope of this PR.

Reply:
Yeah, that came up in head as well, I didn't wanted to change the original Checkbox though because it's not part of this PR. But I'll probably change it to use `target` instead of `currentTarget`.

I could also change `handleChange` of this component to use `target` instead of `currentTarget` which is probably a bit safer.